ext/ffi/lib.rs

```diff
@@ -780,8 +800,17 @@ fn make_sync_fn<'s>(
 
   #[cfg(not(target_os = "windows"))]
   let mut fast_allocations: Option<*mut ()> = None;
+
+  #[cfg(target_os = "linux")]
+  let fastapi_is_available = *is_fastapi_available();
+  #[cfg(all(not(target_os = "linux"), not(target_os = "windows")))]
+  let fastapi_is_available = true;
```

Comment:
It would be better if these conditional compilations were inside the `is_fastapi_available()` rather than conditionally defining this function...

Reply:
I changed it, do you like it better that way ?